site-templates/pages/what-is-new-in-v3.gomd

```diff
@@ -29,11 +29,11 @@ The main differences can be grouped in 3 sections:
     <br /><br >
     Please note the following V3 REST APIs that still use the <b>PUT</b> verb to make a change:
     <ul>
-      <li>Create Asset Upload: <pre>/api/v3/workspaces/{workspace}/assets/uploads/{id}/error</pre></li>
-      <li>Progress Asset Upload to processing: <pre>/api/v3/workspaces/{workspace}/assets/uploads/{id}</pre></li>
-      <li>Update the Member's Role:<pre>PUT /api/v3/organizations/{organizationId}/members/{memberId}/role</pre></li>
-      <li>Update Avatar: <pre>PUT /api/v3/users/me/avatar</pre></li>
-      <li>Favourite a workspace: <pre>PUT /api/v3/users/me/favorites/workspaces/{workspaceId}</pre></li>
+      <li>Create Asset Upload: <pre>PUT /v3/workspaces/{workspace}/assets/uploads/{id}/error</pre></li>
```

Comment:
Why does "Create" correspond to "PUT"?

Reply:
Just following the documentation:
![image](https://user-images.githubusercontent.com/51762982/109233440-bcad7100-777e-11eb-8fff-a3b03dc32eb3.png)

Reply:
yes, does not make sense, but that is how the documentation states it.

Reply:
Ah, it's part of the error endpoint. I suppose it's updating the error message for failed uploads.